features/steps/steps.py

```diff
@@ -12,17 +12,23 @@
 CONTAINER_PREFIX = "behave-test-"
 
 
-@given("a `{series}` lxd container with ubuntu-advantage-tools installed")
-def given_a_lxd_container(context, series):
+@given("a `{series}` machine with ubuntu-advantage-tools installed")
+def given_a_machine(context, series):
     if series in context.reuse_container:
```

Comment:
I don't know if we should do this, but maybe we should check if the machine the user is trying to reuse fits the testing it is trying to use. For example, we try to reuse a container on a vm test. The test will fail, but we could alert sooner that the types are not matched and just skip or abort the test.

Reply:
@lucasmoura you are correct that our tests do that, and they currently do. In before_scenario and now in before_feature we check via a call to features/environment:_should_skip_tags which checks whether a conflicting ```@uses or @series``` tag is declared